marimbabot_vision/scripts/flatten_whiteboard_data.py

```diff
@@ -0,0 +1,25 @@
+import os
+import tqdm
+from multiprocessing import Pool
+import re
+from shutil import copyfile
+
+NUM_WORKER = 30
+INPUT_DIR = 'data'
+OUTPUT_DIR = 'data_wb'
+NAME_PREFIX = 'staff_1'
+
+def main(sample):
+    sample_folder = os.path.join(INPUT_DIR, sample)
+    for real_im in [i for i in os.listdir(sample_folder) if os.path.isfile(os.path.join(sample_folder,i)) and 'real' in i]:
```

Comment:
This works. Another way would be the usage of `glob` which might be a bit more consice.